explainer-server/public/stylesheets/main.css

```diff
@@ -212,5 +212,11 @@ body {
   border-left: 1px solid #bdbdbd;
 }
 
+#top-toolbar__word-count-wrapper {
+  border-left: 1px solid #bdbdbd;
+  padding:0px 15px 0px 15px;

```

Comment:
This can be done in shorthand, the `bottom` and `left` will inherit from `top` and `right` respectively. Also, you shouldn't use units on 0 values as it's not needed.
So you can change this to `padding: 0 15px` for the same effect.